Logging/ARKLogMessage.h

```diff
@@ -27,15 +27,23 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface ARKLogMessage : NSObject <NSCopying, NSSecureCoding>
 
+/// @warning Do not use this initializer.
+/// @raises NSInternalInconsistencyException This is not a valid initializer.
+- (instancetype)init;
```

Comment:
Ah. I see why you did this now. Just mark init and new as NS_UNAVAILABLE to prevent the need for this grossness